activerecord/test/cases/dirty_test.rb

```diff
@@ -671,6 +671,25 @@ def test_datetime_attribute_doesnt_change_if_zone_is_modified_in_string
     assert binary.changed?
   end
 
+  test "changes is correct if override attribute accessor" do
```

Comment:
I'd rewrite this example message as well as the commit message to reference `reader` or `getter` explicitly, since `writter/setter` overrides do not affect Dirty hash negatively.